components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import SchoolCard from '../SchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('SchoolCard', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(
```

Comment:
I like you trying to remove duplicate code with the before each!

You can take it a step farther by defining 

```
const componentInstance = <SchoolCard {...etc} />
```

and using it in `wrapper = shallow(componentInstance)` PLUS `createShallowSnapshotTest(componentInstance)`